### ISSUE TITLE
Document / improve lint rule blacklisting

### ISSUE THREAD

**Issue by xixixao** [hhvm/hhast#256]
To disable a rule the config needs to use the fully namespaced lint rule class name:

```json
{
  "roots": ["bin/", "src/"],
  "builtinLinters": "all",
  "disabledLinters": ["Facebook\\HHAST\\NoStringInterpolationLinter"]
}
```

That's not super user friendly, and not super obvious even reading the getConfig implementation.

**Comment by fredemmott** [comment]
Yeah, the documentation needs to be improved, but the need for fully-qualified names will be remaining: there are multiple users with custom lint rules, including variants of standard lint rules, and don’t want to make non-default linters included with HHAST “special” compared to non-included linters - this applies here because the blacklist syntax should be consistent with the whitelist syntax

**Comment by fredemmott** [comment]
There are other issues making non-included lingers less desirable, but those are bugs/areas for improvement, e.g #13

**Comment by lexidor** [comment]
Fixed by #520, which documents the hhast-lint.json file properties.

Particularly of note: https://github.com/hhvm/hhast/blob/14c9fc7f7bdd140c3999af24349d80dc847bb828/docs/configuring-hhast-lint.md?plain=1#L202-L213
